### ISSUE TITLE
Add aria-busy to html tag during transitions

### ISSUE THREAD

**Issue by hirasso** [swup/swup#466]
I think it would be a good idea to add `aria-busy="true"` to the `html` tag during transitions, just like [hotwire/turbo](https://turbo.hotwired.dev/) does it. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy

Not sure if this should be implemented in the core or in the [Accessibility Plugin](https://github.com/swup/a11y-plugin). What do you guys think?

**Before creating this issue, did you think of...**:
- [x] Have you checked closed issues for similar/related problems.
- [x] Have you provided all helpful information available?

**Comment by daun** [comment]
Yes! It's been on my list for the accessibility plugin, but I haven't come around to it if you want to create a PR.

I do think it's nice for all a11y stuff to be in the plugin to keep everything tidy. Maybe we should add a strong suggestion to use it in the docs?

**Comment by hirasso** [comment]
This was added to `master` and will be part of the next release.
